docs/source/test_network.md

```diff
@@ -0,0 +1,578 @@
+Creating volume "net_peer0.org2.example.com" with default driver
+Creating peer0.org1.example.com ... done
+Creating orderer.example.com    ... done
+Creating peer0.org2.example.com ... done
+Creating cli                    ... done
+CONTAINER ID        IMAGE                               COMMAND             CREATED             STATUS                  PORTS                    NAMES
+3d944d39fab3        hyperledger/fabric-tools:latest     "/bin/bash"         2 seconds ago       Up Less than a second                            cli
+280eaff2c840        hyperledger/fabric-peer:latest      "peer node start"   5 seconds ago       Up 1 second             0.0.0.0:9051->9051/tcp   peer0.org2.example.com
+0907470d42ee        hyperledger/fabric-orderer:latest   "orderer"           5 seconds ago       Up 1 second             0.0.0.0:7050->7050/tcp   orderer.example.com
+f1edf38980b5        hyperledger/fabric-peer:latest      "peer node start"   5 seconds ago       Up 2 seconds            0.0.0.0:7051->7051/tcp   peer0.org1.example.com
+```
+
+If you don't get this result, jump down to [Troubleshooting](#troubleshooting)
+for help on what might have gone wrong.
+
+### The components of the test network
+
+When the network running on your machine, you should take some time to examine
+what was created by the script. Run the following command to list all of Docker
+containers that are running on your machine. You should see the four nodes that
+were created by the `network.sh` script:
+```
+docker ps -a
+```
+
+Each node and user that interacts with a Fabric network needs to belong to an
+organization that is a network member. The group of organizations that are
+members of a Fabric network are often referred to as the consortium. The test
+network has two consortium members, `org1.example.com` and `org2.example.com`.
+The network also includes one orderer organization, `example.com`, who maintains
+the ordering service of the network.
+
+Peer are the fundamental components of any Fabric network. Peers store the
+blockchain ledger and validate transactions before they are committed to the
+ledger. Peers run the smart contracts that encode the business logic that is used
+to manage the assets on the blockchain ledger.
+
+Every peer in the network needs to belong to a member of the consortium. In the
+test network, each organization operates one peer each, `peer0.org1.example.com`
+and `peer0.org2.example.com`.
+
+Every Fabric network also includes an ordering service that is the common biding
+of the network. While peers validate transactions and add blocks of transactions
+to the blockchain ledger, they do not decide on the order of transactions or
+include them into new blocks. On a distributed network, peers may be running far
+away from each other and not have a common view of when a transaction was created.
+Coming to consensus on the order of transactions is a costly process that would
+create overhead for the peers.
+
```

Comment:
I don't think "common binding" is a helpful way of describing what the ordering service does. Policies are more accurately described as common binding. Ordering nodes are sort of simple things. Not as simple as a CA, but pretty simple. They perform a degree of validation (not chaincode validation, obviously) and create blocks. They don't, as far as I know, "decide" the order of the transactions in the blocks. I think that's actually determined by the order in which it receives transactions and the block cutting parameters There's also application logic that factors in here around when a transaction begins its life (enabling the peers to bundle, if it needs to, transactions it's sending to the ordering service). I used to know this stuff a little better but I'm fuzzier on it now.

Reply:
In general I think this section should be linking to other docs where some of these components and processes are conceptually described in more detail. This test network and the doc are starting points for people to branch off. We need to provide some of those links here imo.

Reply:
"Every Fabric network also includes an ordering service", works well, so I will cut out the common binding. I want to cut down on the number of links, (see the current BYFN), but linking out to the concept topics by making the first "peer" and "ordering Service"
 links seems like a good idea.